tests/SitemapControllerTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace cebe\luya\sitemap\tests;
+
+use luya\testsuite\cases\WebApplicationTestCase;
+use cebe\luya\sitemap\Module;
+use cebe\luya\sitemap\controllers\SitemapController;
+use luya\testsuite\fixtures\ActiveRecordFixture;
+use luya\cms\models\NavItem;
+use luya\cms\models\Nav;
+use luya\admin\models\Lang;
+
+class SitemapControllerTest extends WebApplicationTestCase
+{
+    public function getConfigArray()
+    {
+        return [
+           'id' => 'mytestapp',
+           'basePath' => dirname(__DIR__),
+           'aliases' => [
+               'runtime' => dirname(__DIR__) . '/tests/runtime',
+           ],
+           'modules' => [
+               'cms' => 'luya\cms\frontend\Module',
+           ],
+           'components' => [
+                'db' => [
+                    'class' => 'yii\db\Connection',
+                    'dsn' => 'sqlite::memory:',
+                ],
+                'request' => [
+                    'hostInfo' => 'https://luya.io',
+                ]
+            ]
+        ];
+    }
+    
+    public function testIgnoreHiddenModuleProperty()
+    {
+        $module = new Module('sitemap');
+        $module->module = $this->app;
+        $module->withHidden = false;
+        
+        $this->prepareBasicTableStructureAndData();
+        
+        $ctrl = new SitemapController('sitemap', $module);
+        $response = $ctrl->actionIndex();
+        $stream = $response->stream;
+        
+        // @TODO: Does not deliver the xml stream content ...
+        $content = stream_get_contents($stream[0]);
```

Comment:
@cebe Do you know why the yii\web\Response stream can not be converted into a string in oder to compare?